integrations-and-supported-tools/great-expectations/notebooks/Neptune_Great_Expectations.ipynb

```diff
@@ -391,8 +437,15 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "for idx, result in enumerate(results_dict[\"results\"]):\n",
-    "    run[\"gx/validations/json/results\"][idx] = result"
+    "for idx, result in enumerate(results.describe_dict()[\"validation_results\"][0][\"expectations\"]):\n",
+    "    run[\"gx/validations/json/results\"][idx] = stringify_unsupported(result)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Since the results contains lists, we use [`stringify_unsupported()`](https://docs.neptune.ai/api/utils/#stringify_unsupported) to parse them into strings."
```

Comment:
```suggestion
    "Since the results contains lists, use [`stringify_unsupported()`](https://docs.neptune.ai/api/utils/#stringify_unsupported) to parse them into strings."
```